pkg/client/logging/initcontext.go

```diff
@@ -63,3 +66,34 @@ func InitContext(ctx context.Context, name string) (context.Context, error) {
 	}
 	return ctx, nil
 }
+
+func SummarizeLog(ctx context.Context, name string) (string, error) {
+	dir, err := filelocation.AppUserLogDir(ctx)
+	if err != nil {
+		return "", err
+	}
+
+	filename := filepath.Join(dir, name+".log")
+	file, err := os.Open(filename)
+	if err != nil {
+		return "", err
+	}
+	scanner := bufio.NewScanner(file)
+
+	errors := []string{}
```

Comment:
Why not just use an integer?

Reply:
Only because I expect to make it fancier it a later PR. ;-)